spec/features/admin/councils/show_spec.rb

```diff
@@ -15,14 +15,6 @@
     end
   end
 
-  context 'anonymous' do
```

Comment:
same with these -- we had a mistake once where some pages were correctly blocking anonymous users -- these tests can save us.

Reply:
👍